### PR TITLE
Expansion of explanations in messaging

### PR DESCRIPTION
**Pull request by Seanstoppable**

### CODE REVIEW

File: recommendations/Messaging.md [hunk 1]
```diff
@@ -3,14 +3,22 @@
...
 
 ## Assess
 
   - [Amazon Kinesis](http://aws.amazon.com/kinesis/)
+  
+    Potential replacement for Kafka. Note, there are a number of limits currently in Kinesis.
+
   - [Amazon SNS](http://aws.amazon.com/sns/)
+    
+    Potential replacement for rabbit, where messages to SNS can be hooked up to multiple SQS topics for consumption.
```
Comment (andreyk0) [inline recommendations/Messaging.md]: Probably worth noting that it's for 'lossy broadcast', if someone is not familiar with our historical use of rabbit.

Comment (Seanstoppable) [inline recommendations/Messaging.md]: I am currently under the impression that SNS should reliably delivery to SQS. And SQS is 'at least once' delivery. If correct, 'lossy broadcast' would be a result of programming/setup issues vs architecture issues.

Comment (andreyk0) [inline recommendations/Messaging.md]: :+1: Ah, cool!

**Comment by gheine** [comment]
Voted For: 8, Against: 0, Abstained: 0

**Pull request merged**